tla/consensus/MCccfraft.cfg

```diff
@@ -75,8 +75,6 @@ INVARIANTS
     TypeInv
     MonoTermInv
     MonoLogInv
-    LogConfigurationConsistentInv
```

Comment:
@heidihoward you were very right about this, because I had an [off-by-one error](https://github.com/microsoft/CCF/pull/5828/files#diff-f72a34d202ec743abd93cf9ebdfa3b2d5ef7bba962620e7015972ff8f445112bR791) in `ChangeConfigurationInt` that it did catch.

Reply:
Plus an unnecessary config in the joined initial state https://github.com/microsoft/CCF/pull/5828/commits/bb454656728aec3df031fba4882e524827c3d8b5.